qemu/tests/systemtap_tracing.py

```diff
@@ -65,11 +65,18 @@ def create_patterns_reg(trace_key):
                 cmd_type = "bash"
                 exec_cmds = cmd
             for cmd_exec in exec_cmds.split(";"):
-                error.context("Execute %s cmd '%s'" %
-                              (cmd_type, cmd_exec), logging.info)
                 if cmd_type == "monitor":
+                    monitor_type = vm.monitor.protocol
+                    monitor_cmd = "%s_%s" % (monitor_type, cmd_exec)
```

Comment:
Hi @xutian why use prefix here? Normally in autotest we use suffix for different parameters.